src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,12 +2,24 @@
 
 ## Unreleased
 
+## 1.6.0
+
+Released 2023-Sep-05
+
 * Increased the character limit of the Meter instrument name from 63 to 255.
   ([#4774](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4774))
 
 * Update default size for `SimpleExemplarReservoir` to `1`.
   ([#4803](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4803))
 
+* Update Metrics SDK to override the default histogram bounds for a set of
+  well-known histogram metrics from AspNetCore and Http runtime and their
```

Comment:
```suggestion
  well-known histogram metrics for ASP.NET Core and HttpClient
```

Reply:
I think it is ok to skip the instrumentation library part (We will add details on instrumentation libraries side on how to enable these bounds).